code/replication/nml_parseutil.py

```diff
@@ -78,6 +78,8 @@ def create_article_map(directory_path, output_csv_name, k_hours=measure_const.NU
     
     for file in os.listdir(directory):
```

Comment:
os.listdir does not provide the files in alphabetical order. Please check and ensure the files would be read sequentially if named news_2001.nml, news_2002.nml, news_2003.nml, etc.